action.yml

```
@@ -348,7 +348,7 @@ runs:
       shell: bash
     - uses: actions/cache@v4
       with:
-        key: cvmfs-apt-cache-${{ steps.lsb-release.outputs.id-release }}-${{ steps.lsb-release.outputs.arch }}-${{ hashFiles('action.yml') }}
+        key: cvmfs-apt-cache-${{ steps.lsb-release.outputs.id-release }}-${{ steps.lsb-release.outputs.arch }}
```

```suggestion
        key: cvmfs-apt-cache-v1-${{ steps.lsb-release.outputs.id-release }}-${{ steps.lsb-release.outputs.arch }}
```
My proposal: That gives the maintainers a field to manually  invalidate the cache if there's some changes to the apt installations. I don't think it makes sense to tie it to the version of our action, we may want to add changes that don't affect the cache at all.

---

I _think_ you may be able to tweak `$GITHUB_ACTION_PATH` to set a value in your `lsb_release` step that is linked to the action version (tag or commit). I will test it out.